Resources/Prototypes/_LostParadise/Entities/Objects/Consumable/Drinks/drinks.yml

```
@@ -0,0 +1,31 @@
+        - ReagentId: LPPMiruvor
+          Quantity: 30
+  - type: Icon
+    sprite: _LostParadise/Objects/Consumable/Drinks/miruvor.rsi
+    state: icon
+
+- type: entity
+  parent: DrinkGlass
+  id: DrinkLPPCargoGargleBlasterGlass
+  suffix: cargo gargle blaster
+  components:
+  - type: SolutionContainerManager
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)